packages/salesforcedx-vscode-lightning/test/vscode-integration/navigation.test.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as path from 'path';
+import { Location, Position, commands, window, workspace } from 'vscode';
+import URI from 'vscode-uri';
+
+describe('Aura Definition Linking', function() {
+  let auraDir: string;
+  let lwcDir: string;
+
+  before(async function() {
+    auraDir = path.join(
+      workspace.workspaceFolders![0].uri.fsPath,
+      'force-app',
+      'main',
+      'default',
+      'aura'
+    );
+
+    lwcDir = path.join(
+      workspace.workspaceFolders![0].uri.fsPath,
+      'force-app',
+      'main',
+      'default',
+      'lwc'
+    );
+
+    await new Promise(r => setTimeout(r, 1500));
+  });
+
+  afterEach(async function() {
+    await commands.executeCommand('workbench.action.closeActiveEditor');
+  });
+
+  it('Should provide navigation to a selected LWC tag', async function() {
```

Comment:
are other test cases coming later? like navigation to a selected Aura tag?

Reply:
Yes. I need to create/find a sample with Aura to Aura linking.